ci-scripts/bump-python-version.sh

```diff
@@ -5,4 +5,4 @@ source "${SCRIPT_PATH}"/common.sh
 
 version=$1
 echo "new version ${version}"
-echo "__version__ = '${version}'" >lambda_handlers/version.py
+echo "\"\"\"Release version number.\"\"\"\n__version__ = '${version}'  # noqa" >> lambda_handlers/version.py
```

Comment:
```bash
(
    cat <<EOF
"""Release version number."""
__version__ = '${version}'  # noqa
EOF
) >lambda_handlers/version.py
```

`>` otherwise content will be appended rather than replaced

Reply:
done